### ISSUE TITLE
how do we link existing ssl into whmcs

### ISSUE THREAD

**Issue by hendranata** [gogetssl/whmcs-addon#83]
helo
how do we link exiting active ssl into whmcs module?
if that is possible would be good..
thanks

**Comment by vlad-evg** [comment]
What do you mean? Please describe

**Comment by DennisSkov** [comment]
I think he wants to import SSL certificates from GGSSL to WHMCS. This can be done using Addons > SSLCENTER WHMCS > Import SSL Order.

**Comment by vlad-evg** [comment]
Yes, sounds like that :) https://github.com/gogetssl/whmcs-addon/wiki/Manual-Import

Thank you Dennis for helping ;)

**Comment by hendranata** [comment]
sure.
but import only specific user only.. not all

**Comment by DennisSkov** [comment]
You'd need to assign the certificate to a client. It makes no sense having the certificate in WHMCS if it's not associated with a product.

**Comment by hendranata** [comment]
i mean if customer A has got ssl cert from gogetssl under my account. (assume customer A does not have ssl product in my WHMCS, and i want to add him/her to my whmcs)

but i want to link customer A to be having access from my whmcs.. how do we linked it ?

**Comment by DennisSkov** [comment]
You'll need to add the client to WHMCS first. After that, use the importer.

**Comment by DennisSkov** [comment]
That is the order ID from GGSSL.
